editors/emacs/Makefile

```diff
@@ -23,6 +23,14 @@ $(NAME)-$(VERSION).tar: $(SRC)
 	cp *.el "$(NAME)-$(VERSION)"
 	tar -cf "$(NAME)-$(VERSION)".tar "$(NAME)-$(VERSION)"
 
+.PHONY: lambdapi-mode-pkg.el
+lambdapi-mode-pkg.el: $(LAMBDAPI_FILE)
+	$(eval description := $(shell cat $< | grep ";;; $(LAMBDAPI_FILE) ---" | sed "s/;;; $(LAMBDAPI_FILE) --- \(.*\)-\*- lexical-binding:.*/\1/"))
```

Comment:
Can be simplified:
```head -n1 $< | sed "s/;;; $(LAMBDAPI_FILE) --- \(.*\)-\*- lexical-binding:.*/\1/"```
But is it worth doing this as the description is not going to change?
All the more so since it is used for local testing only.